tests/util.test.js

```diff
@@ -13,6 +13,13 @@ const tempPath = path.join(__dirname, 'TEMP')
 process.env['RUNNER_TEMP'] = tempPath
 process.env['RUNNER_TOOL_CACHE'] = cachePath
 
+// Personal Access Token to perform tests with.
+// https://github.com/settings/tokens/new
+const token = process.env['TEST_GITHUB_TOKEN'];
+if (!token) {
+    throw new Error('Set `export TEST_GITHUB_TOKEN="<your token>"` to run tests');
+}
+
```

Comment:
This change will break the CI build - as there is no env var "TEST_GITHUB_TOKEN" defined.
For the unit tests, no real token is required (hence I used `my_test_token`), because all the requests to the github api are mocked using nock. Since you added a new API call, we likely just need to mock this as well.

It seems, you updated the mock - so I guess this change here is not needed anymore.

Reply:
Ah, understood. I'll revert this change. Thanks